code/controllers/subsystem/mobs.dm

```diff
@@ -26,7 +37,24 @@ SUBSYSTEM_DEF(mobs)
 /datum/controller/subsystem/mobs/fire(resumed = 0)
 	var/seconds = wait * 0.1
 	if (!resumed)
-		src.currentrun = GLOB.mob_living_list.Copy()
+		if(crate == 1)
+			var/most = -1
+			var/least = 999999
```

Comment:
maybe this so it's immediatly clear that it's just some high number without concrete meaning.
```suggestion
			var/least = INFINITY
```